crates/editor/src/actions.rs

```diff
@@ -13,6 +13,12 @@ pub struct SelectPrevious {
     pub replace_newest: bool,
 }
 
+#[derive(PartialEq, Clone, Deserialize, Default)]
+pub struct MoveToBeginningOfLine {
+    #[serde(default)]
```

Comment:
This changes the current default, which corresponded to `stop_at_soft_wraps: true`, let's keep it unchanged instead and use https://github.com/zed-industries/zed/blob/8a79535b84dfee2f88a3ad865465893ec3cdf540/crates/language/src/language_settings.rs#L421 or similar idea for that.

Reply:
Is the current behavior (i.e., stopping at soft-wraps) desired, or is it a bug? Defaulting to `false` would be a change to existing behavior, but maybe that is fine if the behavior is unintended?

If the current behavior is intended, then this PR becomes more of an enhancement ("allow configuring stop-at-soft-wrap when moving to line ends") than a bug fix. In that case I should update the PR title and release notes. I don't mind either way -- let me know what you think!

Reply:
Yeah, I'd say this looks more like an improvement and the old behavior is intended (at least, for some group of people).